frameworks/elastic/tests/test_overlay.py

```diff
@@ -55,7 +55,8 @@ def fun():
         observed_name = doc["_source"]["name"]
         return observed_name == "Loren"
 
-    return shakedown.wait_for(fun, timeout_seconds=config.DEFAULT_ELASTIC_TIMEOUT)
+    return shakedown.wait_for(fun, timeout_seconds=config.DEFAULT_ELASTIC_TIMEOUT,
```

Comment:
This test actually does nothing at the moment.... It returns True or False...  which means that failures are a success.